submissions/cammer87.md

```diff
@@ -0,0 +1,17 @@
+* submitter: @cammer87
+* statement: I hereby agree that the facts I have submitted are my own, and that I grant a license for Pryaxis Limited, or designated entities by Pryaxis Limited, the ability to use this data in a way to promote future Pryaxis services.
+
+----
+
+* @IsleRoyaleNationalPark is the least visited national park in the United States.
+* The wolf population of @IsleRoyaleNationalPark is dying off due to genetic inbreeding.
+* @Isle Royale National Park is only accessible by boat or seaplane.
+* The moose of @IsleRoyaleNationalPark have no major predator now that the wolves are declining.
+* The National Park Service will decide at the end of 2017 whether or not to introduce new wolves into @IsleRoyaleNationalPark.
+
+Caleb:
```

Comment:
Thank you for human facts! Not many people go this route, but I appreciate these 300% as much as other facts because they give insights into how I would want this system to work. Thank you!